src/main/java/com/google/sps/servlets/FeedServlet.java

```diff
@@ -1,15 +1,47 @@
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key; 
+import com.google.appengine.api.datastore.KeyFactory; 
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import com.google.sps.data.Parse;
+import com.google.sps.data.QueryDatastore;
+import com.google.sps.data.Post;
+import com.google.sps.data.Representative;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 @WebServlet ("/feed")
-public class FeedServlet extends HttpServlet{
+public class FeedServlet extends HttpServlet {
     @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String repName = request.getParameter("repName");
+        Entity repEntity = QueryDatastore.queryForRepresentative(repName); 
+        if (repEntity == null) {
+            System.out.println("Unable to query representative from datastore"); 
```

Comment:
Does this print statement actually go anywhere useful (ie you can see it later). If not, remove it.